### DISCUSSION TITLE
Storybook for component libraries

### DISCUSSION THREAD

**Discussion by jherdman** [storybookjs/storybook#26371] [Help] [answered]
### Summary

I'm looking into incorporating Storybook into some component libraries. One is using React, and the other is using Ruby, both currently use Storybook v6 and leverage Storybook Server. How does one leverage Storybook v7? Storybook Server doesn't appear to have any associate documentation.

**Comment by Marklb** [comment, marked as answer]
I have never used `@storybook/server`, so I don't know much about it or how well it is supported, but from a brief look through the repository the most up to date documentation that I saw seems to be the README for [@storybook/server-webpack5](https://www.npmjs.com/package/@storybook/server-webpack5)

As for code references:

- [server-kitchen-sink](https://github.com/storybookjs/storybook/tree/next/test-storybooks/server-kitchen-sink)
- [@storybook/server-webpack5](https://github.com/storybookjs/storybook/tree/next/code/frameworks/server-webpack5)
- [@storybook/server](https://github.com/storybookjs/storybook/tree/next/code/renderers/server)
